model/client4.go

```diff
@@ -518,5 +522,18 @@ func (c *Client4) GetPostsForChannel(channelId string, page, perPage int, etag s
 	}
 }
 
+func (c *Client4) GetPing() (bool, *Response) {
```

Comment:
I will change the ping response to ReturnStatusOK() because the ping endpoint should really only do the ping function: check if the server is up.